framework/decode/vulkan_replay_consumer_base.cpp

```diff
@@ -2056,6 +2056,10 @@ void VulkanReplayConsumerBase::InitializeScreenshotHandler()
 
     if (!options_.screenshot_dir.empty())
     {
+        if(!util::filepath::IsDirectory(options_.screenshot_dir))
```

Comment:
What will happen if the thing named already exists and is a file?  I think it may be useful to look at the example in `tools/extract/main.cpp`.

Reply:
yes!

Reply:
May I squash & merge now?

Reply:
Yes, squash then "Rebase and Merge".  I think you might have to squash it locally yourself first.  IDK if GitHub does that but I always manually do it.

Reply:
My final commit was already squashed locally.